runner/src/mill/runner/MillMain.scala

```diff
@@ -305,75 +350,52 @@ object MillMain {
 
   def runBspSession(
       streams0: SystemStreams,
-      runMillBootstrap: (Option[RunnerState], SystemStreams) => RunnerState
+      logStreams: SystemStreams,
```

Comment:
Can we please add some scaladoc stating the purpose of `streams0` and `logStreams` including their difference? This is also useful for devs.

Reply:
Maybe rename `streams0` to `bspProtocolStreams`

Reply:
I should include some hints when it is ok to send something to `bspStreams.err` (as it appears in the BSP client log output).

Reply:
Should be fixed